backend/src/yeomanui.ts

```diff
@@ -73,7 +76,28 @@ export class YeomanUI {
     // on the other hand, we never look for newly installed generators...
 
     const promise: Promise<IPrompt> = new Promise(resolve => {
+      const cwd = path.join(os.homedir(), "projects");
       const env: Environment.Options = Environment.createEnv();
+      const envGetNpmPaths: Function = env.getNpmPaths;
```

Comment:
lint error:
ERROR: src/yeomanui.ts:81:29 - Don't use 'Function' as a type. Avoid using the `Function` type. Prefer a specific function type, like `() => void`.